configure_user/scripts/add_user.sh

```diff
@@ -5,6 +5,15 @@ set -e
 # shellcheck disable=SC2154
 token="$(echo "$iam_token" | awk '{print $2}')"
 
+# validate the token
+token_validation="$(curl -d "token=$token" "https://iam.cloud.ibm.com/identity/introspect?pretty=true" | jq -r .active)"
```

Comment:
not an issue for initial release, but having the iam url here is going to be problematic for customers who want to access iam through vpe/cse